docker-compose.yml

```diff
@@ -8,6 +8,8 @@ services:
     ports:
       - "3306:3306"
       - "33060:33060"
+    security_opt:
```

Comment:
Why is this?

Reply:
Hum probably a tab fix by the IDE. I'll remove it from the commit.

Reply:
Fixed